### PR TITLE
fix(CI): lints and jest config

### PR DESCRIPTION
**Pull request by gitdallas**
ready.

fixed some linting issues, removed the build:brand workflow as it doesn't seem to apply to new ui, made some config changes for jest to be happy, removed esm imports that conflict with the cjs used in test environment

### CODE REVIEW

**Comment by gitdallas** [comment]
looks like i also inadvertently created this branch off nicole's pr, so that commit is in here too 😆

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -49,11 +49,6 @@ jobs:
         run: |
           tar -cf ${UPSTREAM_TAR} dist
           gzip -f --best ${UPSTREAM_TAR}
-
```
Comment (nicolearagao) [inline .github/workflows/integration.yml:52]: I am not sure about this. @bruno-fs @infinitewarp do you remember what this step did ?

File: src/pages/scans/useScansQuery.ts [hunk 2]
```diff
@@ -24,7 +20,7 @@ export const useScansQuery = ({
     baseUrl: process.env.REACT_APP_SCANS_SERVICE,
     columnOrderMap: {
       id: 'id',
-      most_recent: 'most_recent_connect_scan__start_time',
+      most_recent: 'most_recent_connect_scan__start_time'
```
Comment (bruno-fs) [inline src/pages/scans/useScansQuery.ts:23]: sometimes what the linters prefers burns my eyes 🤣

**Review by bruno-fs — APPROVED** [review]
LGTM

**Pull request merged**